packages/cli/src/WorkflowHelpers.ts

```diff
@@ -175,8 +175,7 @@ export async function executeErrorWorkflow(
 		try {
 			await PermissionChecker.checkSubworkflowExecutePolicy(
 				workflowInstance,
-				runningUser.id,
-				workflowErrorData.workflow.id,
+				workflowErrorData.workflow.id!,
```

Comment:
In here we could use `workflowInstance.getNode(workflowErrorData.execution?.lastNodeExecuted);` if available as it should contain the last executed node.